validations/src/tests.rs

```diff
@@ -2326,6 +2326,7 @@ fn test_empty_commit(c_tx: &CommitTransaction) -> Result<(), failure::Error> {
     let collateral_minimum = 1;
     let collateral_age = 1;
     let block_number = 0;
+    let minimum_reppoe_difficulty = 1;
```

Comment:
Smart to avoid chainging the tests! :smile:

Reply:
The idea is to keep old tests working exactly like before and add new tests checking the new formula, but I don't know how to do that in a maintainable way.